ios/engine/KMEI/KeymanEngine/Classes/Manager.swift

```diff
@@ -180,11 +180,12 @@ public class Manager: NSObject, HTTPDownloadDelegate, UIGestureRecognizerDelegat
     hold.delegate = self
     keymanWeb.view.addGestureRecognizer(hold)
 
-    reachability = Reachability(hostName: keymanHostName)
-    NotificationCenter.default.addObserver(self, selector: #selector(self.reachabilityChanged),
-                                           name: .reachabilityChanged, object: reachability)
-    reachability.startNotifier()
-
+    if(!Util.isSystemKeyboard) {
+      reachability = Reachability(hostName: keymanHostName)
```

Comment:
On the original PR, you left this initialization at line 183 before the utility check.